fdbserver/ConfigNode.actor.cpp

```diff
@@ -132,6 +135,27 @@ class ConfigNodeImpl {
 	Counter getGenerationRequests;
 	Future<Void> logger;
 
+	ACTOR static Future<size_t> getCoordinatorsHash(ConfigNodeImpl* self) {
+		state size_t coordinatorsHash = 0;
+		Optional<Value> value = wait(self->kvStore->readValue(coordinatorsHashKey));
+		if (value.present()) {
+			coordinatorsHash = BinaryReader::fromStringRef<size_t>(value.get(), IncludeVersion());
+		} else {
+			self->kvStore->set(
+			    KeyValueRef(coordinatorsHashKey, BinaryWriter::toValue(coordinatorsHash, IncludeVersion())));
+			wait(self->kvStore->commit());
+		}
+		return coordinatorsHash;
+	}
+
+	ACTOR static Future<Optional<size_t>> getLocked(ConfigNodeImpl* self) {
```

Comment:
I think here it would be helpful to add a comment that `size_t` is a coordinator hash in the return type.